docs/cn/Configuring-Alluxio-with-GCS.md

```diff
@@ -27,12 +27,11 @@ priority: 0
 $ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
 ```
 
-为了配置Alluxio以使用GCS作为其底层文件系统，需要修改`conf/alluxio-env.sh`文件。第一个需要修改的地方就是指定一个已有的bucket以及bucket中的目录作为底层文件系统。你可以通过向`conf/alluxio-env.sh`文件添加以下代码来指定：
+你需要配置Alluxio，以使用GCS作为其底层文件系统。第一个需要修改的地方就是指定一个已有的bucket以及bucket中的目录作为底层文件系统。你可以通过向`conf/alluxio-env.sh`文件添加以下代码来指定：
```

Comment:
`conf/alluxio-site.properties` , not `conf/alluxio-env.sh`

Reply:
I have changed it.